src/main/java/org/jenkinsci/test/acceptance/docker/DockerContainerHolder.java

```diff
@@ -36,7 +38,7 @@
      */
     @Inject(optional = true)
     @Named("dockerPortOffset")
-    private int portOffset = 0;
+    private Integer portOffset;
```

Comment:
@mafraba Indeed `port 0` means:

```
/* 
 * Obtain a reference to a local port for the given sock, if snum is zero it means select any available local port.
 */
```

Negative ports are not valid. Maybe you can use `-1`?